lib/http-context.js

```diff
@@ -379,6 +406,11 @@ function toXML(input) {
  */
 
 HttpContext.prototype.done = function() {
+  if (this.method.stream) {
+    // the stream has already been setup
+    return;
+  }
+
```

Comment:
We need to handle errors and I can't reference the error here (unless there is a `ctx.error` property??).